src/main/java/org/alliancegenome/curation_api/model/document/es/ExpressionDetailDocument.java

```diff
@@ -0,0 +1,38 @@
+package org.alliancegenome.curation_api.model.document.es;
+
+import java.io.Serializable;
+import java.util.List;
+
+import org.alliancegenome.curation_api.model.entities.CrossReference;
+import org.alliancegenome.curation_api.model.entities.Gene;
+import org.alliancegenome.curation_api.model.entities.InformationContentEntity;
+import org.alliancegenome.curation_api.model.entities.Organization;
+import org.alliancegenome.curation_api.model.entities.ontology.MMOTerm;
+import org.alliancegenome.curation_api.model.entities.ontology.StageTerm;
+import org.jetbrains.annotations.NotNull;
+import com.fasterxml.jackson.annotation.JsonView;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@EqualsAndHashCode(callSuper = true)
+@Data
+@JsonView({org.alliancegenome.curation_api.view.View.ExpressionDetailDocument.class})
```

Comment:
Can we import the package and just have the class name here for consistency/readability